### PR TITLE
Use DacFx for SQL Server 2022

### PR DESCRIPTION
**Pull request by ErikEJ**
Use microsoft.sqlpackage on container

### CODE REVIEW

File: src/DacpacTool/PackageDeployer.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public PackageDeployer(IConsole console)
             _console = console ?? throw new ArgumentNullException(nameof(console));
         }
 
-        public SqlConnectionStringBuilder ConnectionStringBuilder { get; private set; } = new SqlConnectionStringBuilder();
+        public SqlConnectionStringBuilder ConnectionStringBuilder { get; private set; } = new SqlConnectionStringBuilder { Encrypt = false };
```
Comment (jmezach) [inline src/DacpacTool/PackageDeployer.cs:21]: Hmm, do we want to hardcode this? I think we should at least make this configurable.

Comment (ErikEJ) [reply src/DacpacTool/PackageDeployer.cs:21]: Is this not just setting the default value?

Comment (ErikEJ) [reply src/DacpacTool/PackageDeployer.cs:21]: we have not set "trust server certificate" before either. And this makes publish work just like before - for local dev publishing

Comment (jmezach) [reply src/DacpacTool/PackageDeployer.cs:21]: Yeah, I guess it is fine as a default, but then we should at least document it I think. Perhaps there is someone out there who does want to encrypt the connection to his/her development SQL Server and now they don't have a way of doing that. They can make it configurable if they want, but then at least they know it is happening ;).

Comment (ErikEJ) [reply src/DacpacTool/PackageDeployer.cs:21]: How about adding it to DeployOptions and default to false ?

public bool Encrypt { get; set; }

Comment (jmezach) [reply src/DacpacTool/PackageDeployer.cs]: Yeah, that will work :).

**Review by jmezach — APPROVED** [review]

**Review by jeffrosenberg — APPROVED** [review]

**Comment by ErikEJ** [comment]
@jmezach Is it OK for me to merge this now? (or do you need to do something behind the scenes) ?

**Comment by jmezach** [comment]
No, I think we can merge this. After that I can do a release.

**Pull request merged**